packages/runtime-common/realm.ts

```diff
@@ -2298,6 +2300,75 @@ export class Realm {
     });
   }
 
+  private async getCardDef(
+    request: Request,
+    requestContext: RequestContext,
+  ): Promise<Response> {
+    let href = new URL(request.url).search.slice(1);
+    let payload = parseQuery(href);
+    if (!payload.codeRef) {
+      return badRequest({
+        message: `The request body is missing the coderef parameter`,
```

Comment:
```suggestion
        message: `The request body is missing the codeRef parameter`,
```

The message should match the expected name, I think, same for below